systemvm/debian/opt/cloud/bin/cs/CsDhcp.py

```diff
@@ -49,15 +49,21 @@ def process(self):
             self.add(self.dbag[item])
         self.write_hosts()
 
-        if self.cloud.is_changed():
-            self.delete_leases()
-
         self.configure_server()
 
-        restart_dnsmasq = self.conf.commit()
-        self.cloud.commit()
+        restart_dnsmasq = False
+
+        if self.conf.commit():
+            restart_dnsmasq = True
+
+        if self.cloud.commit():
```

Comment:
repeat 2 times commit, why?

Reply:
@resmo The commit is called on two different files, it's the same old code refactored differently.

Reply:
I see conf.commit and cloud.commit. Those are two different objects, @resmo . How is this suspect?

Reply:
I missed that. never mind.